Source/src/modules/ModuleSceneManager.cpp

```diff
@@ -290,6 +290,9 @@ void Hachiko::ModuleSceneManager::ChangeMainScene(Scene* new_scene)
         main_scene->SetCullingCamera(main_scene->GetMainCamera());
     }
 #endif // PLAY_MODE
+
+    // REBUILD NAVMESH
+    App->navigation->RebuildCurrentNavmesh(main_scene);
```

Comment:
This should not be needed since it is already managed on LoadScene. I fixed the issues we were having, and ths would make the navmesh to be build in runtime whenever we change a scene if in not mistaken

Reply:
Nah man it's another issue, if i delte this line each time the scene is paused and played again you need to rebuild manualy the scene by config, try it yourself and you will see that is a pain